SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -508,3 +535,43 @@ void USpatialClassInfoManager::QuitGame()
 	FGenericPlatformMisc::RequestExit(false);
 #endif
 }
+
+Worker_ComponentId USpatialClassInfoManager::ComputeActorInterestComponentId(const AActor* Actor) const
+{
+	check(Actor);
+	const AActor* ActorForRelevancy = Actor;
+	// bAlwaysRelevant takes precedence over bNetUseOwnerRelevancy - see AActor::IsNetRelevantFor
```

Comment:
If always relevant takes precedence over anything else, why isn't that check first, and then this check would not need the `!ActorForRelevancy->bAlwaysRelevant` below?

Reply:
Not exactly sure I follow. bAlwaysRelevant is checked on the current actor (actor updated within the loop). If it's ever true, we know we can abort the loop and return `ALWAYS_RELEVANT_COMPONENT_ID`

Reply:
ignore me, I thought it was an if and not a while loop